M1_test.py

```diff
@@ -0,0 +1,161 @@
+from uni_henn.service import *
+from seal import *
+from torchvision import datasets, transforms
+import numpy as np
+import torch
+import h5py, os
+import time
+import math
+
+# public_key.save('key/public_key')
+# secret_key.save('key/secret_key')
+# galois_key.save('key/galois_key')
+# relin_keys.save('key/relin_keys')
+
+import torch.nn.functional as F
+import torch.optim as optim
+
+class CNN(torch.nn.Module):
+    def __init__(self, hidden=64, output=10):
+        super(CNN, self).__init__()
+        self.Conv1 = torch.nn.Conv2d(in_channels=1, out_channels=8, kernel_size=4, stride=3, padding=0)
+        self.FC1 = torch.nn.Linear(9 * 9 * 8, 64)
+        self.FC2 = torch.nn.Linear(64, 10)

```

Comment:
기본값으로 할당해놓은거 쓰면 좋겠네요

```suggestion
        self.FC1 = torch.nn.Linear(9 * 9 * 8, hidden)
        self.FC2 = torch.nn.Linear(hidden, output)

```